mne/cov.py

```diff
@@ -842,6 +836,19 @@ def _unpack_epochs(epochs):
     return out
 
 
+def _check_scalings_user(scalings):
+    if isinstance(scalings, dict):
+        for k, v in scalings.items():
+            if k not in ('mag', 'grad', 'eeg'):
```

Comment:
Depends on whether or not they are sufficiently oversampled by the sensor array. I suppose we could add them